connectors/src/connectors/snowflake/index.ts

```diff
@@ -117,7 +118,33 @@ export class SnowflakeConnectorManager extends BaseConnectorManager<null> {
   }
 
   async resume(): Promise<Result<undefined, Error>> {
-    throw new Error("Method resume not implemented.");
+    const connector = await ConnectorResource.fetchById(this.connectorId);
+
+    if (!connector) {
+      logger.error(
+        {
+          connectorId: this.connectorId,
+        },
+        "Snowflake connector not found."
+      );
+      return new Err(new Error("Connector not found"));
+    }
+
+    const dataSourceConfig = dataSourceConfigFromConnector(connector);
+    try {
+      await launchSnowflakeSyncWorkflow(connector.id);
```

Comment:
this returns a result. We don't want to try catch here but instead handle the `isErr()` case

Reply:
Updated, fyi, I copy-pasted from [there](https://github.com/dust-tt/dust/blob/e454acc5f1d544812ba8a4e05e767011ddaecb2a/connectors/src/connectors/notion/index.ts#L244) (will update as well when refactoring notion).

Reply:
Notion one doesn't return a result (was created before we started doing this). Should probably be updated at some point indeed 👍